configs/config.json.example

```diff
@@ -12,6 +12,7 @@
         "config": {
           "enabled": false,
           "master": null,
+          password: null,

```

Comment:
"password" needs to be in quotes.

And is the approval system not there to enforce a four-eye-principle?